sdk/storage/Azure.Storage.DataMovement/src/LocalFileStorageResource.cs

```diff
@@ -216,6 +221,8 @@ protected internal override Task CopyBlockFromUriAsync(
         /// <returns>Returns the properties of the Local File Storage Resource. See <see cref="StorageResourceItemProperties"/></returns>
         protected internal override Task<StorageResourceItemProperties> GetPropertiesAsync(CancellationToken cancellationToken = default)
         {
+            CancellationHelper.ThrowIfCancellationRequested(cancellationToken);
```

Comment:
this implementation of the base method is completely sync. This can probably be reverted.

Reply:
Yeah I saw that the implementation is sync. Just seems weird to me that we are passing in a CancellationToken but not using it at all. Is that fine?